feg/protos/mconfig/mconfigs.proto

```diff
@@ -56,8 +56,12 @@ message S6aConfig {
 }
 
 message VirtualApnRule {
+    // Regex to match APN values
```

Comment:
Here and elsewhere -- prefer golang-style doc comments, i.e. start the comment with `// apn_filter is a regex ...`